openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -88,7 +87,7 @@ <h3 class="peer-assessment__display__title">
 
             <div class="step__actions">
                 <div class="message message--inline message--error message--error-server" tabindex="-1">
-                    <h3 class="message__title">{% trans "We could not submit your assessment" %}</h3>
+                    <h5 class="message__title">{% trans "We could not submit your assessment" %}</h5>
```

Comment:
I have the same question in this template as I did below with Training Assessments. I would expect that headers within the assessment form shown after "Assessment # of x" would be children instead of peers (relative to the header "Assessment # of x"). @cptvitamin, what do you think?

![image](https://cloud.githubusercontent.com/assets/484484/20941747/be37ad48-bbc5-11e6-8bdb-9350c16c34e7.png)

![image](https://cloud.githubusercontent.com/assets/484484/20941698/6c926b36-bbc5-11e6-9ee6-e47382a2f75e.png)